### DISCUSSION TITLE
Can I override layouts?

### DISCUSSION THREAD

**Discussion by ZekeriyaAY** [jpanther/congo#33] [Q&A] [answered]
Hello, I have manually installed the theme before. I'm reinstalling with the Hugo module to get theme updates. 
Can I override layouts? As with my old files https://github.com/ZekeriyaAY/zekeriyaay.github.io/tree/main/themes/congo/layouts

**Comment by jpanther** [comment, marked as answer]
Yes, you can override any of the layouts but it looks like you're attempting to manually change the theme files. As per the [Hugo docs](https://gohugo.io/templates/partials/#partial-template-lookup-order), layout overrides should go in your base project directory. If you place your files in `layouts/` then they should override just as they would if you installed the theme manually.

**Reply by ZekeriyaAY** [reply]
Does it cause problems when you update the theme?

**Reply by jpanther** [reply]
No, whatever overrides you have in place will take precedence over the theme. This is simply the standard Hugo behaviour that works the same way with any theme.
